libs/auth-api-lib/src/lib/delegations/models/delegation.model.ts

```diff
@@ -78,17 +79,13 @@ export class Delegation extends Model<
     }
 
     // 2. Find items with value in the array
-    const arrDates = this.delegationScopes
+    const dates = (this.delegationScopes
       ?.filter((x) => x.validTo !== null && x.validTo !== undefined)
-      .map((x) => x.validTo) as Array<Date>
-    if (arrDates && arrDates.length > 0) {
-      // Return the max value
-      return arrDates.reduce((a, b) => {
-        return a > b ? a : b
-      })
-    }
+      .map((x) => x.validTo) || []) as Array<Date>
 
-    return undefined
+    // Return the max value
+    const m = max(dates)
+    return m
```

Comment:
~ Redundant variable assignment.
```suggestion
    return max(dates)
```

Reply:
:sweat_smile: Thanks, forgot to fix after debugging.